### ISSUE TITLE
Automatic deletion of merged branches?

### ISSUE THREAD

**Issue by vreuter** [pepkit/peppy#427]
A nice setting to consider turning on is automatic deletion of branches once merged. What do you think @nsheff ?

**Comment by nsheff** [comment]
Yes that's great -- except the dev branch usually I don't mind keeping open, since development typically continues there right away, but the rest of the branches absolutely, that would be nice! I'm often frustrated by people leaving feature branches behind and I have to ask them to remove them if they're finished.

I probably do the same thing ...
